actions/addNewProject.ts

```diff
@@ -3,5 +3,5 @@
 import { IAddProjectForm } from '@/types/forms';
 
 export const addNewProject = async (data: IAddProjectForm) => {
-  console.log(data);
+  console.log(data.fullName);
 };
```

Comment:
You can leave it as `console.log(data)`

Reply:
sure my bad Is there anything else that i need to change ?

Reply:
No thats all